explorer/tsconfig.json

```diff
@@ -4,6 +4,7 @@
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
     "skipLibCheck": true,
+    "downlevelIteration": true,
```

Comment:
Useful for iterating through `Map`

Reply:
Ah, I ran into this last week. Ended up going with `for(let entry of Array.from(map.entries())) {`. A welcome change.